calico/felix/config.py

```diff
@@ -184,6 +184,12 @@ def __init__(self, config_path):
                            "none", sources=[ENV, FILE])
         self.add_parameter("EtcdCaFile", "Path to etcd CA certificate file",
                            "none", sources=[ENV, FILE])
+        self.add_parameter("EtcdEndpoints", "Comma separated list of etcd "
```

Comment:
I don't really understand - these are the config options.  The scheme is used internally; we strip the schemes out of the endpoints.  It's all slightly icky, but I'm not convinced there's much improvement available without investing a whole bunch of work.